Server crash when Energized Power Auto Crafter checks TransmuteWithoutMaterialRecipe

## Description

Using an Energized Power Auto Crafter in FTB Unstable 6 causes the dedicated server to crash while the crafter scans crafting recipes.

The crash occurs inside `TransmuteWithoutMaterialRecipe.matches()` from Little Big Redstone.

The recipe calls:

```java
input.getItem(0)
```

but in this case the `CraftingInput` passed by Energized Power contains no items, resulting in:

```text
java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
```

This crashes the server as a ticking block entity.

## Environment

* Modpack: FTB Unstable 6
* Modpack version: 1.9.0
* Dedicated server
* Little Big Redstone: `1.9.8-alpha-26.1.2`
* Energized Power: `3.0.0+26.1.x-neoforge`
* Refined Storage: `3.2.1`

## Relevant stack trace

```text
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
    at java.base/java.util.ImmutableCollections$ListN.get(ImmutableCollections.java:725)
    at net.minecraft.world.item.crafting.CraftingInput.getItem(CraftingInput.java:86)
    at net.swedz.little_big_redstone.recipe.TransmuteWithoutMaterialRecipe.matches(TransmuteWithoutMaterialRecipe.java:63)
    at net.swedz.little_big_redstone.recipe.TransmuteWithoutMaterialRecipe.matches(TransmuteWithoutMaterialRecipe.java:25)
    at me.jddev0.ep.block.entity.AbstractAutoCrafterBlockEntity.lambda$getRecipesFor$1(AbstractAutoCrafterBlockEntity.java:804)
    at me.jddev0.ep.block.entity.AbstractAutoCrafterBlockEntity.getRecipesFor(AbstractAutoCrafterBlockEntity.java:806)
    at me.jddev0.ep.block.entity.AbstractAutoCrafterBlockEntity.getRecipeFor(AbstractAutoCrafterBlockEntity.java:810)
    at me.jddev0.ep.block.entity.AbstractAutoCrafterBlockEntity.updateRecipe(AbstractAutoCrafterBlockEntity.java:470)
    at me.jddev0.ep.block.entity.AbstractAutoCrafterBlockEntity.onTickStart(AbstractAutoCrafterBlockEntity.java:363)
    at me.jddev0.ep.block.entity.base.WorkerMachineBlockEntity.tick(WorkerMachineBlockEntity.java:122)
```

## Expected behavior

`TransmuteWithoutMaterialRecipe.matches()` should safely return `false` when the supplied crafting input is empty instead of accessing item index 0.

For example, it seems like the method may need to guard against an empty input before calling `input.getItem(0)`.

## Actual behavior

When Energized Power checks available crafting recipes with an empty crafting input, `TransmuteWithoutMaterialRecipe.matches()` accesses index 0 and throws an `ArrayIndexOutOfBoundsException`, crashing the dedicated server.

## Reproduction

1. Run FTB Unstable 6 v1.9.0 on a dedicated server.
2. Place/use an Energized Power Auto Crafter.
3. Allow the Auto Crafter to update/check its crafting recipe.
4. During recipe matching it evaluates Little Big Redstone's `TransmuteWithoutMaterialRecipe`.
5. The server crashes with a ticking block entity exception.

[crash-2026-08-28_13.52.00-server.txt](https://github.com/user-attachments/files/31553458/crash-2026-08-28_13.52.00-server.txt)

--- a/src/main/java/net/swedz/little_big_redstone/recipe/TransmuteWithoutMaterialRecipe.java
+++ b/src/main/java/net/swedz/little_big_redstone/recipe/TransmuteWithoutMaterialRecipe.java
@@ -60,7 +60,7 @@ public final class TransmuteWithoutMaterialRecipe extends NormalCraftingRecipe
 	@Override
 	public boolean matches(CraftingInput input, Level level)
 	{
-		return ingredient.test(input.getItem(0));
+		return !input.items().isEmpty() && ingredient.test(input.getItem(0));
 	}
 	
 	@Override

